src/views/studio/StudioToolbar/StudioToolbar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styles from "./StudioToolbar.module.scss";
+import BackArrow from "../../../assets/vectors/back-arrow.svg"
+
+interface StudioToolbarProps {
+    title: string
+    onBackPress?: () => void
+}
+
+export const StudioToolbar: React.FC<StudioToolbarProps> = ({title, onBackPress}) => {
+    return (
+        <div className={styles.container}>
+            {onBackPress &&
+                <BackArrow
+                    width={24}
+                    height={24}
+                    className={styles.backArrow}
+                    onClick={() => onBackPress()}/>
+            }
+
+            <span>{title}</span>
+        </div>
+    )
+}
```

Comment:
yeah, about that
![image](https://user-images.githubusercontent.com/32616724/167258624-43a78a97-4bee-49df-a1bd-32e6779e8566.png)

Reply:
fixed too